Patches/Vanilla Weapons Expanded - Frontier/Weapons_Frontier-Laser.xml

```diff
@@ -2,28 +2,206 @@
+      </li>
+
+      <!-- === Salvaged Laser Revolver === -->
+
+      <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+        <defName>VWEFT_Gun_SalvagedLaserRevolver</defName>
+        <statBases>
+          <Mass>1.40</Mass>
+          <Bulk>2.60</Bulk>
+          <SwayFactor>1.33</SwayFactor>
+          <ShotSpread>0.07</ShotSpread>
+          <SightsEfficiency>0.7</SightsEfficiency>
+          <RangedWeapon_Cooldown>0.65</RangedWeapon_Cooldown>
+        </statBases>
+        <Properties>
+          <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+          <hasStandardCommand>True</hasStandardCommand>
+          <defaultProjectile>Bullet_Laser_SalvagedLaserRevolver_HR</defaultProjectile>
+          <warmupTime>0.6</warmupTime>
+          <range>16</range>
+          <soundCast>VWE_UnstableLaserShot_Light</soundCast>
+          <soundCastTail>GunTail_Heavy</soundCastTail>
+          <muzzleFlashScale>9</muzzleFlashScale>
+        </Properties>
+        <AmmoUser>
+          <magazineSize>16</magazineSize>
+          <reloadTime>4</reloadTime>
+          <ammoSet>AmmoSet_SalvagedLaserRevolver_HR</ammoSet>
+        </AmmoUser>
+        <FireModes>
+          <aiAimMode>AimedShot</aiAimMode>
+        </FireModes>
+        <weaponTags>
+          <li>SalvagedLaserGun</li>
+          <li>CE_OneHandedWeapon</li>
+        </weaponTags>
+      </li>
+
+      <!-- === Salvaged Laser Repeater === -->
+ 
+      <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+        <defName>VWEFT_Gun_SalvagedLaserRepeater</defName>
+        <statBases>
+          <Mass>3.60</Mass>
+          <Bulk>10.10</Bulk>
+          <SwayFactor>1.37</SwayFactor>
+          <ShotSpread>0.08</ShotSpread>
+          <SightsEfficiency>1.00</SightsEfficiency>
+          <RangedWeapon_Cooldown>0.55</RangedWeapon_Cooldown>
+        </statBases>
+        <Properties>
+          <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+          <hasStandardCommand>True</hasStandardCommand>
+          <defaultProjectile>Bullet_Laser_SalvagedLaserRepeater_HR</defaultProjectile>
+          <warmupTime>1</warmupTime>
+          <range>62</range> 
+          <soundCast>VWE_UnstableLaserShot_Rifle</soundCast>
+          <soundCastTail>GunTail_Heavy</soundCastTail>
+          <muzzleFlashScale>9</muzzleFlashScale>
+        </Properties>
+        <AmmoUser>
+          <magazineSize>20</magazineSize>
+          <reloadTime>4</reloadTime>
+          <ammoSet>AmmoSet_SalvagedLaserRepeater_HR</ammoSet>
+        </AmmoUser>
+        <FireModes>
+          <aiAimMode>AimedShot</aiAimMode>
+          <aiUseBurstMode>True</aiUseBurstMode>         
+        </FireModes>
+        <weaponTags>
+          <li>SalvagedLaserGun</li>
+        </weaponTags>		
+      </li>
+
+      <!-- === Salvaged Laser Hunting Rifle === -->
+
+      <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+        <defName>VWEFT_Bullet_SalvagedLaserHuntingRifle</defName>
```

Comment:
Patch is targeting the bullet, not the weapon itself.

Reply:
Opps, got sloppy with a find+replace. Thanks.